src/SDK/Client/Platform/broadcastStateTransition.ts

```diff
@@ -23,6 +25,7 @@ export default async function broadcastStateTransition(platform: Platform, state
     // if (!result.isValid()) {
     //     throw new Error(`StateTransition is invalid - ${JSON.stringify(result.getErrors())}`);
     // }
+    const { client: dapiClient } = await client.getDAPIClient();
```

Comment:
This looks strange... It supposed to be sync and respond with DAPI Client instance.

Reply:
The bigger issue is : DAPIClient vs DAPIClientWrapper. 

In DAPIClient, elements are not the object we look for thus the need for the wrapper. 
I agree it's an issue, I propose renaming walletLib.transporters.DAPIClient to DAPIClientWrapper.